spec/helpers/submissions_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
Please refrain from having empty test files. Just delete the file entirely if you're never going to have a submission helper.